FlowCrypt/Common/Taptic.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  Taptic.swift
+//  FlowCrypt
+//
+//  Created by Anton Kharchevskyi on 24.09.2019.
+//  Copyright © 2019 FlowCrypt Limited. All rights reserved.
+//
+
+import UIKit
+
+enum TapTicFeedback {
```

Comment:
What does TapTic mean and what is it used for? Thanks!

Reply:
for iPhones started I think from 8 - they edit taptic feedback when you do some actions. For example on. iPhone X when you tap on button you will have small feedback.